sdk/lib/core/identical.dart

```diff
@@ -5,6 +5,17 @@
 part of dart.core;
 
 /// Check whether two references are to the same object.
+/// ```dart
+///  Car micro = Car(type: 'Micro', color: 'Red');
```

Comment:
Car class definition missing.

Reply:
the whole example could be reformatted to:
`void main() {
  Car micro = Car(type: 'Micro', color: 'Red');
  Car suv = Car(type: 'SUV', color: 'Black');

  var areObjectsIdentical = identical(micro, micro);
  print(areObjectsIdentical); // true

  areObjectsIdentical = identical(micro, suv);
  print(areObjectsIdentical); // false
}

class Car {
  late final type;
  late final color;

  Car({this.type, this.color});
}
`

Reply:
fixed